mise.toml

```diff
@@ -1,6 +1,7 @@
 [tools]
+bun = "1.3"
 dotnet = "9"
 java = "24"
 node = "24"
+pnpm = "10.11.1"
```

Comment:
doesn't it make sense to just use corepack for pnpm since it's already defined in the package.json file? just so we're not maintaining the version in 1 places?